griptape/events/event_bus.py

```diff
@@ -9,8 +9,12 @@
 
 
 @define
-class EventPublisherMixin:
-    event_listeners: list[EventListener] = field(factory=list, kw_only=True)
+class _EventBus:
+    _event_listeners: list[EventListener] = field(factory=list, kw_only=True, alias="_event_listeners")
```

Comment:
https://www.attrs.org/en/stable/init.html#private-attributes-and-aliases

Reply:
This is uglier than just not using attrs imo. Plus it still adds `_event_listeners` as an keyword arg in `init`.

I don't care that much though